packages/app/screens/feed/create group form/CreateGroupFormScreen1.js

```diff
@@ -0,0 +1,236 @@
+    description, setDescription,
+    banner, setBanner,
+    icon, setIcon,
+    resetForm } = useContext(GroupContext);
+
+  // Input Validation
+  const [titleError, setTitleError] = useState("");
+  const [handlerError, setHandlerError] = useState("");
+  const [descriptionError, setDescriptionError] = useState("");
+  const [bannerError, setBannerError] = useState("");
+  const [iconError, setIconError] = useState("");
+
+  useEffect(() => {
+    handleInputValidationScreen1();
+  }, [title, handler, description, banner, icon]);
+
+  function handleInputValidationScreen1() {
+    let isValid = true;
+
+    const titleRegex = /^(?![a-zA-Z0-9]+$).*$/;
+    const handlerRegex = /^(?!(\w{1,15})$).*$/;
+
+    if (title.trim() === ``) {
+      setTitleError(`Please enter a title`);
+      isValid = false;
+    } else if (titleRegex.test(title)) {
+      setTitleError(`Can only contain letters, numbers, and underscores.`);
+      isValid = false;
+    } else {
+      setTitleError(``);
+    }
+
+    if (handler.trim() === ``) {
+      setHandlerError(`Please enter a handler`);
+      isValid = false;
+    } else if (handlerRegex.test(handler)) {
+      setHandlerError(`Can only contain letters, numbers, and underscores.`);
+      isValid = false;
+    } else {
+      setHandlerError(``);
+    }
+
+    if (description.trim() === ``) {
+      console.log(0);
+      setDescriptionError(`Please enter a description`);
+      isValid = false;
+    } else {
+      setDescriptionError(``);
+    }
+
+    if (banner.trim() === ``) {
+      setBannerError(`Please pick a banner image`);
+      isValid = false;
+    } else {
+      setBannerError(``);
+    }
+
+    if (icon.trim() === ``) {
+      setIconError(`Please pick a icon image`);
+      isValid = false;
+    } else {
+      setIconError(``);
+    }
+
+    // isValid = true; //for testing
+    return isValid;
+  }
+
+  const selectBannerImage = async () => {
+    const result = await ImagePicker.launchImageLibraryAsync({
+        mediaTypes: ImagePicker.MediaTypeOptions.Images,
+        allowsEditing: true,
+        aspect: [3, 1],
+        quality: 1,
+        base64: true, // Enable base64 option
+      });
+
+    const { assets } = result;
+
+    if (!result.canceled) {
+        // Process the selected image
+        const { base64 } = assets[0];
+        // console.log("Banner image picked: " + base64);
+        setBanner(base64);
+    }
+  };
+
+  const selectIconImage = async () => {
+    const result = await ImagePicker.launchImageLibraryAsync({
+        mediaTypes: ImagePicker.MediaTypeOptions.Images,
+        allowsEditing: true,
+        aspect: [1, 1],
+        quality: 1,
+        base64: true, // Enable base64 option
+      });
+
+    const { assets } = result;
+    
+    if (!result.canceled) {
+        // Process the selected image
+        const { base64 } = assets[0];
+        // console.log("Icon image picked: " + base64);
+        setIcon(base64);
+    }
+  };
```

Comment:
It may be possible to combine some of the code for selectBannerImage and selectIconImage into one function (like a "selectImage" function)